.github/workflows/main.yaml

```diff
@@ -97,9 +97,29 @@ jobs:
           git config --local user.email "$GITHUB_ACTOR@users.noreply.github.com"
           sudo apt-get install -y clang-format perl || true
           time ./tools/style.sh || true
-          find . -type f \( -name 'DESCRIPTION' -o -name "*.R"\)  ! -path "*/gen-*.*" ! -path "*/lantern.*"  ! -path "*/RcppExports.*" ! -path "./check/*" -printf '%p\n' -exec git add {} \; || true
+          find . -type f \( -name 'DESCRIPTION' -o -name "*.R"\) \
```

Comment:
Can we simplify this and commit all changes at once? I don't think we need separate commits.

Reply:
Yes, we can.. but I think it's good to split R and C/C++ formatting (and cleanup). This way, we can easily debug an error caused by one of them (e.g., a bug in `styler` vs. `clang-format`).

Reply:
Though, I'd move the commits into the script, if that works for you. So, GHA workflow only runs `style.sh`.